Spatial.gd

```diff
@@ -80,6 +85,9 @@ func _ready():
 	OS.set_window_position(Vector2(0,0))
 	set_minimal_mouse_block()
 
+	# Call ready for additional scripts
+	self.file_handler._ready()
```

Comment:
Why are you manually calling `_ready()` which is a function that is called automatically?